### PR TITLE
Releasever for system-upgrade is now set in pre_configure_upgrade

### PR DESCRIPTION
**Pull request by m-blaha**
Also removing now empty file /system-update/.dnf-system-upgrade

### CODE REVIEW

**Comment by j-mracek** [comment]
The code looks nice

**Comment by j-mracek** [comment]
@rh-atomic-bot try

**Comment by j-mracek** [comment]
We will merge it after fc27 release

File: etc/systemd/dnf-system-upgrade.service [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [Unit]
 Description=System Upgrade using DNF
-ConditionPathExists=/system-update/.dnf-system-upgrade
+ConditionPathExists=/system-update
```
Comment (ignatenkobrain) [inline etc/systemd/dnf-system-upgrade.service]: not sure how it would affect pk-offline-update... @hughsie @keszybz ?

**Pull request closed without merging**